tests/foundry/test/AssumeTest.t.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity =0.8.13;
+
+import "forge-std/Test.sol";
+
+contract AssumeTest is Test {
+
+    function test_assume_true(uint256 a, uint256 b) public {
+        vm.assume(a == b);
+        assertEq(a, b);
+    }
+
+    function test_assume_false(uint256 a, uint256 b) public {
+        vm.assume(a != b);
+        assertEq(a, b);
+    }
+
+    function testFail_assume_true(uint256 a, uint256 b) public {
```

Comment:
Does the prefix `testFail_` mean the test is successful if some assertion is violated?

Reply:
Yes, from documentation: `testFail` is the inverse of the test `prefix` - if the function does not revert, the test fails [(see here)](https://book.getfoundry.sh/forge/writing-tests?highlight=testfail#writing-tests)